src/Bedrock.Framework.Experimental/Protocols/RabbitMQ/Methods/ConnectionStart.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.RabbitMQ.Methods
+{
+    public class ConnectionStart : MethodBase, IAmqpMessage
+    {
+        public override byte ClassId => 10;
+        public override byte MethodId => 10;
+
+        public byte VersionMajor { get; private set; }
+        public byte VersionMinor { get; private set; }
+        public Dictionary<string, object> ServerProperties { get; private set; }
+        public ReadOnlyMemory<byte> SecurityMechanims { get; private set; }
+        public ReadOnlyMemory<byte> Locale { get; private set; }
+
+        private ReadOnlySpan<byte> PlainAMQPPlain => new byte[] { (byte)'P', (byte)'L', (byte)'A', (byte)'I', (byte)'N', (byte)' ', (byte)'A', (byte)'M', (byte)'Q', (byte)'P', (byte)'L', (byte)'A', (byte)'I', (byte)'N' };
```

Comment:
With the change to _RabbitMQ_ is this naming still correct?

Reply:
Good question. RabbitMQ implements AMQP 0.9, while RabbitMQ seems the only one which still use this protocol version , but that's the string value returned by RabbitMQ, so the naming is (maybe) not fully inappropriate ?
@davidfowl what do you think about that ?

(sorry for my english :) )